tests/meta-mender-ci/recipes-testing/mender-mock-server/mender-mock-server_1.0.bb

```diff
@@ -0,0 +1,31 @@
+SRC_URI = " \
+    file://private.key \
+    file://mender-mock-server.service \
+    file://mender-mock-server.py \
+"
+LICENSE = "PD"
+LIC_FILES_CHKSUM = " \
+    file://private.key;md5=8a1986b43cf91bb9f47b0af99ad764fd \
+    file://mender-mock-server.service;md5=38751b3b7fe2ab6dbcacaad792128b9a \
+    file://mender-mock-server.py;md5=958275eda2dd2fe87d6ce649fc5599c3 \
```

Comment:
This is a bit overkill as it will trigger a checksum update on every single file change. The common thing is to just provide a license file and use the checksum for that. However, when searching through poky, I learned a new trick: Look at what they did [here](https://github.com/openembedded/meta-openembedded/blob/master/meta-oe/recipes-dbs/sqlite/sqlite_2.8.17.bb#L6), you can use the top license of the python file. For such a small recipe, I think this is a good solution.

Reply:
This is nice!

Reply:
Nice tip, thanks!